frontend/app/utils/date-utils.ts

```diff
@@ -62,9 +62,9 @@ export function toLocaleDateString(date: Date, locale: string) {
   return date.toLocaleDateString(`${locale}-CA`, { year: 'numeric', month: 'long', day: 'numeric' });
 }
 
-export function getAgeFromDateString(dateOfBirth: string) {
-  const parseDateOfBirth = parse(dateOfBirth, 'yyyy-MM-dd', new Date());
-  invariant(isValid(parseDateOfBirth), `date of birth is invalid [${dateOfBirth}]`);
-  invariant(isPast(parseDateOfBirth), `date of birth must be in past [${dateOfBirth}]`);
-  return differenceInYears(new Date(), parseDateOfBirth);
+export function getAgeFromDateString(date: string) {
```

Comment:
There was some discussion around timezones and calculating ages based on NFLD's timezone.  As far as I know it was never resolved, so in my eyes, this is fine to ship and if/when that discussion ever gets resolved, the code can change to incorporate that.

Reply:
This is another discussion are you said.